manifests/cf-manifest/scripts/generate-manifest.sh

```diff
@@ -56,7 +56,6 @@ bosh interpolate \
   --ops-file="${CF_DEPLOYMENT_DIR}/operations/enable-service-discovery.yml" \
   --ops-file="${PROMETHEUS_DEPLOYMENT_DIR}/manifests/operators/cf/add-prometheus-uaa-clients.yml" \
   ${slim_dev_deployment} \
-  --ops-file="${CF_DEPLOYMENT_DIR}/operations/experimental/disable-consul.yml" \
```

Comment:
Now this is gone we can recombine `slim_dev_deployment` into `opsfile_args` (see https://github.com/alphagov/paas-cf/commit/e24a3177#diff-31c9fe7d5b430915c2e9db0a24e645c8)

Reply:
checking :thinking: